templates/python-start-scrape-single-page/src/main.py

```diff
@@ -0,0 +1,24 @@
+from urllib.parse import urljoin
+
+import requests
+from apify import Actor
+from bs4 import BeautifulSoup
+
+async def main():
+    async with Actor:
+        # Read the Actor input
+        actor_input = await Actor.get_input() or {}
+        url = actor_input.get('start_url')
+
+        Actor.log.info(f'Scraping {url}...')
+
+        try:
+            # Fetch the URL using `requests` and parse it using `BeautifulSoup`
+            response = requests.get(url)
+            soup = BeautifulSoup(response.content, 'html.parser')
+
+            # Push the title of the page into the default dataset
+            title = soup.title.string if soup.title else None
```

Comment:
We want to have the logic unified, let's extract all the headings instead of just the title - we want the to user see more items saved to the dataset and to be logged out.

Please check the PR for js https://github.com/apify/actor-templates/pull/150/files and
1. Update the logic to have the same output
2. update the comments to follow the same structure